s64da_benchmark_toolkit/prepare.py

```diff
@@ -178,31 +178,25 @@ def _load_pre_schema(self, conn):
     def _load_license(self, conn):
         try:
             conn.cursor.execute(f'select swarm64da.show_license()')
-        except errors.InvalidSchemaName:
-            print(f'Selected schema is not for S64 DA. Skip checking license.')
-            return
-        except errors.UndefinedFunction:
-            print(f'License check function not required or supported. Skip checking license.')
```

Comment:
There may be the case when the user loaded their own license from a different path. In my opinion we should not let the toolkit fail in that scenario.